src/app/scripts/cac/places/cac-places.js

```diff
@@ -79,77 +79,57 @@ CAC.Places.Places = (function(_, $, moment, Routing, UserPreferences) {
-
+    function getDistancesToPlaces(destinations, exploreLatLng) {
         // prefer to label with just the street address, and fall back to full address
         // for featured places, use the label in the 'name' attribute instead of the address
         var origin = UserPreferences.getPreference('origin');
-        var originLabel = origin.name ? origin.name :
-            (origin.attributes && origin.attributes.StAddr ? origin.attributes.StAddr :
-            UserPreferences.getPreference('originText'));
+        var originLabel;
+
+        if (origin) {
+            originLabel = origin.name ? origin.name :
+                (origin.attributes && origin.attributes.StAddr ? origin.attributes.StAddr :
+                UserPreferences.getPreference('originText'));
+        }
+
+        if (exploreLatLng) {
+            var from = turf.point(exploreLatLng);
+        }
+
+        _.each(destinations, function(destination) {
+            // if origin not set, simply clear the travel times from the destinations
+            if (!from) {
+                destination.distance = undefined;
+                destination.formattedDistance = undefined;
+                destination.originLabel = undefined;
+                return;
+            }
 
-        var promises = _.map(destinations, function(destination) {
-            var dfd = $.Deferred();
             // if already have travel time to destination, skip requerying for it
             if (destination.originLabel === originLabel) {
-                dfd.resolve(destination);
-                return dfd.promise();
+                return;
             }
 
             // read out the location of the destination
             var xCoord = destination.location.x;
             var yCoord = destination.location.y;
 
             if (xCoord && yCoord) {
-                var placeCoords = [yCoord, xCoord];
-                // get travel time to destination and update place card
-                Routing.planTrip(exploreLatLng, placeCoords, date, otpOptions, false)
-                .then(function (itineraries) {
-                    if (itineraries && itineraries.length) {
-                        var itinerary = itineraries[0];
-                        // set properties for travel time to place and the origin label
-                        destination.duration = itinerary.duration;
-                        destination.formattedDuration = itinerary.formattedDuration;
-                        destination.originLabel = originLabel;
-                        dfd.resolve(destination);
-                    }
-                }).fail(function(error) {
-                    console.error('error finding travel time to ' + destination.name);
-                    console.error(error);
-                    dfd.reject(error);
-                });
-            } else {
-                // event without a location
-                dfd.resolve(destination);
+                var to = turf.point([yCoord, xCoord]);
```

Comment:
Should be `[xCoord, yCoord]`.

Reply:
Yes. Was waiting for a good CI build before revisiting.

Reply:
Oops, I forgot to reassign before, otherwise I would have known that :)